include/arc_utilities/arc_helpers.hpp

```diff
@@ -505,99 +480,37 @@ namespace arc_helpers
-            }
-            else if (color_code == 10)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x00, 0x8a, 0x53, alpha);
-            }
-            else if (color_code == 11)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xff, 0x5c, 0x7a, alpha);
-            }
-            else if (color_code == 12)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x53, 0x7a, 0x37, alpha);
-            }
-            else if (color_code == 13)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xff, 0x00, 0x8e, alpha);
-            }
-            else if (color_code == 14)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xb3, 0x51, 0x28, alpha);
-            }
-            else if (color_code == 15)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xf4, 0x00, 0xc8, alpha);
-            }
-            else if (color_code == 16)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x7f, 0x0d, 0x18, alpha);
-            }
-            else if (color_code == 17)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x93, 0x00, 0xaa, alpha);
-            }
-            else if (color_code == 18)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x59, 0x15, 0x33, alpha);
-            }
-            else if (color_code == 19)
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0xf1, 0x13, 0x3a, alpha);
-            }
-            else
-            {
-                return RGBAColorBuilder<ColorType>::MakeFromMixedColors(0x23, 0x16, 0x2c, alpha);
-            }
-        }
-        else
-        {
-            return RGBAColorBuilder<ColorType>::MakeFromFloatColors(0.0f, 0.0f, 0.0f, alpha);
-        }
+        std::map<uint32_t, std::vector<uint8_t>> color_map{
+            {0, {0xff, 0xff, 0xff}},
+            {1, {0xff, 0x00, 0xb3}},
+            {2, {0x80, 0x75, 0x3e}},
+            {3, {0xff, 0x00, 0x68}},
+            {4, {0xa6, 0xd7, 0xbd}},
+            {5, {0xc1, 0x20, 0x00}},
+            {6, {0xce, 0x62, 0xa2}},
+            {7, {0x81, 0x66, 0x70}},
+            {8, {0x00, 0x34, 0x7d}},
+            {9, {0xf6, 0x8e, 0x76}},
+            {10, {0x00, 0x8a, 0x53}},
+            {11, {0xff, 0x5c, 0x7a}},
+            {12, {0x53, 0x7a, 0x37}},
+            {13, {0xff, 0x00, 0x8e}},
+            {14, {0xb3, 0x51, 0x28}},
+            {15, {0xf4, 0x00, 0xc8}},
+            {16, {0x7f, 0x0d, 0x18}},
+            {17, {0x93, 0x00, 0xaa}},
+            {18, {0x59, 0x15, 0x33}},
+            {19, {0xf1, 0x13, 0x3a}},
+            {20, {0x23, 0x16, 0x2c}}
+        };
+
+        if (color_map.count(color_code))
```

Comment:
I'd prefer to change this to the following:
```c++
const auto itr = color_map.find(color_code);
if (itr == color_map.end())
{
    return RGBAColorBuilder<ColorType>::MakeFromFloatColors(0.0f, 0.0f, 0.0f, alpha);
}
const auto& c = itr->second;
return RGBAColorBuilder<ColorType>::MakeFromMixedColors(c[0], c[1], c[2], alpha);
```
In this way the default behaviour is that we find something in the map (otherwise we're not likely using this function), and "unusual" behaviour is in the `if`.